.github/workflows/build.yml

```diff
@@ -16,9 +16,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Checkout code
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
+    - uses: actions/setup-go@v4
+      with:
+        go-version-file: go.mod
```

Comment:
No need to manually bump the Go version in GHA's anymore. 
`setup-go` can now pull from `go.mod` which is the source of truth.
Works well for me in other non-spinnaker repositories.